packages/clay-charts/src/Chart.js

```diff
@@ -11,7 +12,8 @@ import templates from './Chart.soy.js';
 class Chart extends Component {}
 
 Object.assign(Chart.prototype, ChartBase);
-Chart.STATE = ChartBase.STATE;
+Object.assign(Chart.prototype, DataComponent);
```

Comment:
Why not `Object.assign(Chart.prototype, DataComponent, ChartBase);` all in one line?

Reply:
Yes.👍